client/lib/simple-payments/utils.js

```diff
@@ -5,3 +5,8 @@ import { SIMPLE_PAYMENTS_PRODUCT_POST_TYPE } from 'lib/simple-payments/constants
 
 export const isValidSimplePaymentsProduct = product =>
 	product.type === SIMPLE_PAYMENTS_PRODUCT_POST_TYPE && product.status === 'publish';
+
+export const decodeProductAttribute = productAttribute =>
+	productAttribute.replace( /(?:&#(\d+);)|(&amp;)/g, ( match, charCode, amp ) =>
```

Comment:
There is some documentation to understand what is happening here? I mean for instance a known method or whatever?
The only thing I can clearly see there is a person winking at me. :-p 
![image](https://user-images.githubusercontent.com/77539/29050359-c903615c-7bb1-11e7-86f3-472e66bf56d2.png)

Feel free to ignore this comment.